### ISSUE TITLE
Migration?

### ISSUE THREAD

**Issue by fabien-sanglier-ibm** [SoftwareAG/webmethods-helm-charts#113]
Team, any plan to move this to [IBM](https://github.com/ibm/) now webMEthods is a IBM product?
I also have a few helm charts that I'd like to contribute (eg. https://github.com/sag-gov-integration-unit/webmethods-helm-charts), but thought it'd be best to see if there are any plans for this to move first...
I can also help moving this if desired.

**Comment by cospeedster** [comment]
We've got the same question here.

**Comment by cospeedster** [comment]
@fabien-sanglier-ibm Do you have news regarding this?

**Comment by fabien-sanglier-ibm** [comment]
no news... maybe we just do it. I can crate the new repo and migrate this one there.

**Comment by thomas-2020** [comment]
Hi Fabien,
excuse me for my question, but can you explain why we should move to `sag-gov-integration-unit`.

**Comment by fabien-sanglier-ibm** [comment]
Hey, you may have misread... the thread / issue here is to move this repo to IBM Public Github (https://github.com/IBM)... since this repo is for webMethods and as such should be probably better placed there at this point.

For the "sag-gov-integration-unit" charts that I have, that's frankly another topic altogether...
Once this project is migrated to IBM public GitHub, I would possibly provide PRs to this project if/as needed... but again, not the primary topic of this thread.

Does that answer your question?

**Comment by thomas-2020** [comment]
Do you know or have any contact with knowledge how to move or copy GitHub repository (with all assets and permissions)? Additional, we need an Internet DNS entry for `https://open-source.softwareag.com/webmethods-helm-charts/charts` for the new target repo, e.g. `https://open-source.ibm.com/webmethods-helm-charts/charts` .

**Comment by MarcFriedhoff** [comment]
Hi, sorry for the late reply on this. We will transfer the entire repository to the public IBM github soon (still in 2024). @fabien-sanglier-ibm I would recommend to wait with any PRs until then. We will close this repo here then. @thomas-2020 I will contact you separately.

**Comment by MarcFriedhoff** [comment]
@cospeedster, @fabien-sanglier-ibm, @mathieucarbou, @thomas-2020, I have created a new repo here: [https://github.com/IBM/webmethods-helm-charts](https://github.com/IBM/webmethods-helm-charts) - all commits until December 6th, main and gh-pages branch have been transferred. We will close this repo on January 31st 2025. Unfortunately I could not simply clone the entire repository as I also needed to adjust some references and some legal disclaimers, licenses and copyright remarks.

**Comment by mathieucarbou** [comment]
@mobasherul : FYI

@MarcFriedhoff : I guess the opened issues / PR have to be re-created then ?

**Comment by cospeedster** [comment]
@MarcFriedhoff maybe you can leave a note in the readme and archive this repo.

**Comment by cospeedster** [comment]
@MarcFriedhoff I didn't see you do just that.

**Comment by MarcFriedhoff** [comment]
> @mobasherul : FYI
> 
> @MarcFriedhoff : I guess the opened issues / PR have to be re-created then ?

Yes, I could not see how to transfer them. There is currently only one PR pending.



> @MarcFriedhoff maybe you can leave a note in the readme and archive this repo.

I did.
